packages/cf-deploy-config-writer/src/translations/cf-deploy-config-writer.i18n.json

```
@@ -1,9 +1,11 @@
 {
     "debug": {
         "logError": "{{method}} error found {{ error }}",
-        "mtaLoaded": "MTA {{ type }} loaded",
+        "mtaLoaded": "MTA {{ type }} loaded, size {{ size }}",
         "ui5YamlDoesNotExist": "File ui5.yaml does not exist in the project",
         "capMtaCreated": "CAP MTA Configuration created",
+        "capMtaUpdated": "CAP MTA Configuration updated",
+        "capGenerationCompleted": "CAP generation completed",
         "mtaCreated": "MTA Configuration created {{mtaPath}}.",
         "mtaSaved": "MTA Configuration has been saved.",
         "mtaSavedFailed": "MTA saved failed with error {{- error }}"
```

```suggestion
        "mtaSavedFailed": "Saving the MTA configuration failed with the error: {{- error }}."
```
What should do to try to resolve this? View the logs? Try again?